samples/save_and_load.exs

```diff
@@ -0,0 +1,18 @@
+list = Enum.to_list(1..10_000)
+map_fun = fn(i) -> [i, i * i] end
+
+Benchee.run(%{
+  "flat_map"    => fn -> Enum.flat_map(list, map_fun) end,
+  "map.flatten" => fn -> list |> Enum.map(map_fun) |> List.flatten end
+},
+  time: 5,
+  save: [file: "save.benchee", tag: "first-try"]
+)
+
+Benchee.run(%{
+  "flat_map"    => fn -> Enum.flat_map(list, map_fun) end,
+  "map.flatten" => fn -> list |> Enum.map(map_fun) |> List.flatten end
+},
+  time: 5,
+  load: "save_first-try.benchee"
```

Comment:
hm what do you mean? Like `:path` ? I planned on doing that but in the end there was no other option I wanted to pass to `load` so I thought it might be nicer. You're right the other option might be more consistent.

Or do you also mean passing `tag` again? I can see how it _might_ be useful to "retag" all loaded scenarios but thought I'd rather avoid it. Not sure.

Reply:
I think it would be helpful for users if they could do this:

```elixir
list = Enum.to_list(1..10_000)
map_fun = fn(i) -> [i, i * i] end

Benchee.run(%{
  "flat_map"    => fn -> Enum.flat_map(list, map_fun) end,
  "map.flatten" => fn -> list |> Enum.map(map_fun) |> List.flatten end
},
  time: 5,
  save: [file: "save.benchee", tag: "first-try"]
)

Benchee.run(%{
  "flat_map"    => fn -> Enum.flat_map(list, map_fun) end,
  "map.flatten" => fn -> list |> Enum.map(map_fun) |> List.flatten end
},
  time: 5,
  load: [file: "save.benchee", tag: "first-try"]
)
```

Otherwise, we're sort of asking them to either 1) Do the conversion of what the file would be named in their head, or 2) Check their filesystem to find the right file they want to load. Do you think that might be confusing?

Reply:
oh, I wanted to remove the whole "interleave the tag into the filename" feature as a whole, as I figured it'd be confusing. So no matter the tag, if you say `file/path: "save.benchee"` it'll land in `save.benchee` and NOT `save_tag.benchee` - what do you think?

Reply:
Yeah, that'll work fine. I think as long as whatever we pass to `save` can also be passed to `load`, then that should work out. I would assume that for most users they'd be saving and then loading right away, so making that change easy for them would be helpful. They the only thing they need to do for that functionality is to replace `save:` with `load:` and they're good to go.

Reply:
Hm I would still cut out the `tag` part as we don't need it, but I think I won't validate it so it won't do harm. Will implement it some time!